src/webcontents-handler.js

```diff
@@ -126,7 +126,30 @@ function onLinkContextMenu(ev, params) {
 }
 
 function _CutCopyPasteSelectContextMenus(params) {
-    return [{
+    const options = [];
+
+    if (params.misspelledWord) {
+        params.dictionarySuggestions.forEach(word => {
+            options.push({
+                label: word,
+                click: (menuItem, browserWindow) => {
+                    browserWindow.webContents.replaceMisspelling(word);
```

Comment:
I guess this knows what to replace by what the user has highlighted since it's an editing command? The docs seem pretty scarce - either way it look like you're doing the correct thing.

Reply:
indeed, and empirically it does work. When you right click a misspelt word it highlights the entire word before showing the context menu